www/include/configuration/configServers/popup/remote.yaml

```diff
@@ -0,0 +1,35 @@
+name: gorgoned-__SERVERNAME__
+description: Configuration for remote server __SERVERNAME__
+gorgone:
+  gorgonecore:
+    id: __SERVERID__
+    external_com_type: tcp
+    external_com_path: "*:__GORGONEPORT_"
+    authorized_clients: __THUMBPRINT__
+    privkey: "/var/lib/centreon-gorgone/.keys/rsakey.priv.pem"
+    pubkey: "/var/lib/centreon-gorgone/.keys/rsakey.pub.pem"
+  modules:
+    - name: action
+      package: gorgone::modules::core::action::hooks
+      enable: true
+
+    - name: nodes
+      package: gorgone::modules::centreon::nodes::hooks
+      enable: true
+
+    - name: proxy
+      package: gorgone::modules::core::proxy::hooks
+      enable: true
+
+    - name: legacycmd
+      package: gorgone::modules::centreon::legacycmd::hooks
+      enable: true
+      cmd_file: "/var/lib/centreon/centcore.cmd"
```

Comment:
is it possible to choose another path (eg : source installation script on a poller)